components/CardCalendar/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,1156 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+
+import "../index.scss";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS/Navigation/Card Calendar"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Card Calendar" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <div class="denhaag-card-calendar__wrapper">
+      <div class="denhaag-card-calendar__item">
+        <div class="denhaag-card-calendar__item__header">
```

Comment:
Zie BEM convention mijn comment in index.scss en onderstaande waar meer dan 1 niveau de `__` gebruikt wordt.